refactor: Update multiple ctors warning with a space and grammar

srawlins opened the pull request:
refactor: Update multiple ctors warning with a space and grammar

Add a space before the word "using" and make spelling of
constructor/ctor consistent

tbosch commented:
Hi, this looks good. Could you sign the CLA?

srawlins commented:
Thanks @tbosch , signed.

tbosch commented:
@srawlins Sorry, I was late with reviewing this change. Could you rebase ontop of master?

srawlins committed:
refactor: Update multiple ctors warning with a space and grammar

Add a space before the word "using" and make spelling of
constructor/ctor consistent

srawlins commented:
OK all patched and merged and rebased and squashed and up to date and CI looks good! :smile:

mary-poppins commented:
Merging PR #3923 on behalf of @mhevery to branch [presubmit-mhevery-pr-3923](https://github.com/angular/angular/tree/presubmit-mhevery-pr-3923).

srawlins commented:
Ping @mhevery , did that presubmit branch go well?

mary-poppins commented:
Merging PR #3923 on behalf of @mhevery to branch [presubmit-mhevery-pr-3923](https://github.com/angular/angular/tree/presubmit-mhevery-pr-3923).

mhevery commented:
This needs to be rebased on latest master, @srawlins can you do that? Many thanks

srawlins commented:
Thanks @mhevery; rebased. Looking good.

mary-poppins commented:
Merging PR #3923 on behalf of @mhevery to branch [presubmit-mhevery-pr-3923](https://github.com/angular/angular/tree/presubmit-mhevery-pr-3923).
